tests/testthat/_snaps/proj-desc.md

```diff
@@ -10,3 +10,10 @@
     Message
       v Adding 'bravo' to Config/Needs/foofy
 
+# proj_desc_field_update() works with multiple values
+
+    Code
+      proj_desc_field_update("Config/Needs/foofy", c("alfa", "bravo"), append = TRUE)
+    Message
+      v Adding 'alfa', 'bravo' to Config/Needs/foofy
```

Comment:
I'm not sure this would be worth the fiddliness. But arguably the set logic should be applied *before* we message. Maybe just worth keeping in mind if we touch this again.